drivers/spi/spi_andes_atcspi200.c

```diff
@@ -12,14 +12,39 @@
 
 typedef void (*atcspi200_cfg_func_t)(void);
 
+#ifdef CONFIG_ANDES_SPI_DMA_MODE
+
+#define ANDES_SPI_DMA_ERROR_FLAG 0x01
+#define ANDES_SPI_DMA_RX_DONE_FLAG 0x02
+#define ANDES_SPI_DMA_TX_DONE_FLAG 0x04
+#define ANDES_SPI_DMA_DONE_FLAG	\
+	(ANDES_SPI_DMA_RX_DONE_FLAG | ANDES_SPI_DMA_TX_DONE_FLAG)
+
+struct stream {
+	const struct device *dma_dev;
+	uint32_t channel;
+	uint32_t block_idx;
+	struct dma_config dma_cfg;
+	struct dma_block_config dma_blk_cfg;
+	struct dma_block_config chain_block[MAX_CHAIN_SIZE];
+	uint8_t priority;
+	bool src_addr_increment;
+	bool dst_addr_increment;
+};
+#endif
+
 struct spi_atcspi200_data {
+	const struct device *self_dev;
```

Comment:
This shouldn't be needed

Reply:
OK, yes, that is more appropriate.
I have update the source code.

Thank you.